Squirrely-Ducks/bangazon-sprint-3
# added some minor styling changes to appeal to the masses

## DESCRIPTION
What did you do? 
added some minor details to styling
Why did you do it?
a bit easier on the eyes
## STEPS TO TEST
1. pull down branch
2. check if styles are pleasurable to the optic nerves
3. Test functionality
	* 
	* 
	* 

## REFERENCE
Does it reference a ticket?
